lib/omniauth/strategies/openid_connect.rb

```diff
@@ -466,6 +467,7 @@ def verify_id_token!(id_token)
 
         decode_id_token(id_token).verify!(issuer: options.issuer,
                                           client_id: client_options.identifier,
+                                          audience: client_options.audience,
```

Comment:
Would it be safer to pass in `audience` only if it is present? I noticed in https://github.com/nov/openid_connect/blob/a3ec9145dcb637e420ebbb9b1858bdadfbe3e575/lib/openid_connect/response_object/id_token.rb#L30 that the logic might behave differently if `audience` were a blank string, for exmaple.

Reply:
Good catch. Seems safer to only pass it if present.

I'll update it and let you know